docs/src/main/paradox/docs/delta/api/files-api.md

```diff
@@ -89,6 +89,62 @@ Request
 Response
 :   @@snip [created-put.json](assets/files/created-put.json)
 
+## Create copy using POST or PUT
+
+Create a file copy based on a source file potentially in a different organization. No `MIME` details are necessary since this is not a file upload. Metadata such as the size and digest of the source file are preserved.
+
+The caller must have the following permissions:
+- `files/read` on the source project.
+- `storages/write` on the storage in the destination project.
```

Comment:
You need the permission defined on the destination storage

Reply:
Thanks found that out during testing as well, gonna rewrite these docs once it's finished